value-processor/src/org/immutables/value/processor/meta/Constitution.java

```diff
@@ -62,7 +63,14 @@ public Generics generics() {
 
 	@Value.Derived
 	public Visibility implementationVisibility() {
-		if (style().visibility() == ImplementationVisibility.PRIVATE
+		ImplementationVisibility visibility;
+		if (!Strings.isNullOrEmpty(style().visibilityString())) {
+			visibility = ImplementationVisibility.valueOf(style().visibilityString());
+		} else {
+			visibility = style().visibility();
```

Comment:
I like and use tabs for indents, but this repo uses 2 spaces indentation. (I've seen some other code accidentally has tabs, but that was sloppiness and exception to the rule)

Reply:
you're right, I'll reformat this file at some point

Reply:
Yep, this file is in need for reformat, but thank you for getting your lines correctly